new/LIT7268WaBaSades.xml

```diff
@@ -0,0 +1,71 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7268WaBaSades" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ወበሳድስ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Wa-ba-sādəs</title>
+                <title xml:lang="en" corresp="#t1">And on the sixth…</title>
+                <title xml:lang="gez" xml:id="t2">ወበሳድስ፡ ወርኅ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t2">Wa-ba-sādəs warḫ</title>
+                <title xml:lang="en" corresp="#t2">And on the sixth month…</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="JK"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Textual arrangement consisting of 
+                    <ref type="work" corresp="LIT3108RepCh388"/>,
+                    verses from <ref type="work" corresp="LIT2713Luke"/> (1:26–38),
+                    and a litany consisting of <foreign xml:lang="gez">ሰላም፡ ለኪ፡</foreign> plus different Marian titles etc.
```

Comment:
maybe better ... and a litany saying ´peace to you´ (ሰላም፡ ለኪ፡) plus different Marian titles etc.